conf/app_vP1.routes

```diff
@@ -0,0 +1,3 @@
+# microservice specific routes
```

Comment:
the name of the file looks a bit funny.
Should we call it app_version_P1.routes instead?

Reply:
I don't mind either way. Unless someone says otherwise I'll rename it to `app_version_P1.routes` and the other one to `app_version_1.routes` for consistency?